website/content/docs/agent/index.mdx

```diff
@@ -379,4 +379,4 @@ template {
 [listener]: /docs/agent#listener-stanza
 [listener_main]: /docs/configuration/listener/tcp
 [winsvc]: /docs/agent/winsvc
-[telemetry]: /docs/configuration/telemetry
\ No newline at end of file
+[telemetry]: /docs/configuration/telemetry
```

Comment:
I couldn't actually tell where this part of the file is on the preview, and it seems like the diff here might be an end of line character or something, so 🤷🏻‍♀️